docs/excel/custom-functions-debugging.md

```diff
@@ -56,7 +59,10 @@ You can use VS Code to debug UI-less custom functions in Office Excel on the des
 ### Start the VS Code debugger
 
 1. Choose **View > Run** or enter **Ctrl+Shift+D** to switch to debug view.
-1. From the Run drop-down menu, choose **Excel Desktop (Custom Functions)**.
+1. From the **Run and Debug** drop-down menu, choose **Excel Desktop (Custom Functions)**.
+
+    :::image type="content" source="../images/custom-functions-run-and-debug-menu.jpg" alt-text="A screenshot showing Excel Desktop (Custom Functions) in the Run and Debug drop-down menu.":::
+
 1. Select **F5** (or select **Run -> Start Debugging** from the menu) to begin debugging. A new Excel workbook will open with your add-in already sideloaded and ready to use.
 
 ### Start debugging
```

Comment:
> I can't verify this section because my breakpoints don't work. They are little white circles labelled "Unbound breakpoint" and they are never hit.

This is a bug that is being worked on.